frontend/src/router/routeHandlers.ts

```diff
@@ -61,6 +63,16 @@ const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
             )
         );
     }
+
+    if (
+        LoginUserState.getUserIdpProvider() == IdpProvider.BCEIDBUSINESS &&
+        LoginUserState.getUserAdminRoleGroups()?.includes(
+            AdminRoleAuthGroup.DelegatedAdmin
+        )
+    ) {
+        showTerms();
```

Comment:
So have you consider the case when user comes back from "Add User Permission page"?